File: Programmers/Lv.0/정사각형으로 만들기/181830.js
function solution(arr) {
    const rownum = arr.length;
    const colnum = arr[0].length;
    const num = Math.max(rownum, colnum);
    
    const answer = new Array(num);
    for (let n=0; n<num; n++) answer[n] = new Array(num).fill(0);
    
    for (let i=0; i<arr.length; i++) {
        for (let j=0; j<arr[i].length; j++) {
            answer[i][j] = arr[i][j];
        }
    }
    
    return answer;
}

File: Programmers/Lv.0/정사각형으로 만들기/181830.java
class Solution {
    public int[][] solution(int[][] arr) {
        int rownum = arr.length;
        int colnum = arr[0].length;
        int num = Math.max(rownum, colnum);
        
        int[][] answer = new int[num][num];
        
        for (int i=0; i<arr.length; i++) {
            for (int j=0; j<arr[i].length; j++) {
                answer[i][j] = arr[i][j];
            }
        }
        
        return answer;
    }
}
